sql/rikai--0.1.sql

```diff
@@ -26,6 +26,27 @@ AS $$
 	return rikai.__version__.version
 $$ LANGUAGE plpython3u;
 
+CREATE FUNCTION ml.python_version()
+    RETURNS TEXT
+AS $$
+    import sys
+    return sys.version
+$$ LANGUAGE plpython3u;
+
+CREATE FUNCTION ml.python_path()
+    RETURNS TEXT
+AS $$
+    import sys
+    return sys.path
+$$ LANGUAGE plpython3u;
+
+CREATE FUNCTION ml.cuda()
+    RETURNS TEXT
```

Comment:
is `torch` has a string of CUDA devices number or sth? or make this a `ml.cuda_available() RETURNS bool`? later we can use this function for some computation / filtering then.

Reply:
i made it a boolean. Added `ml.cuda_info` which returns something like:

```
{"version": "10.2", "seed": 5735813825328988, "device_count": 1, "is_available": true, "allocated": 0, "reserved": 0, "devices": [{"name": "Tesla T4", "total_memory": 15843721216, "processor_count": 40}]}
```